### PR TITLE
Fix a number of typos

### PR DESCRIPTION
**Pull request by pandabadger**
#### Problem
Fix a number of typos

#### Summary of Changes
Mostly in docs and a couple of comments 

Fixes #

### CODE REVIEW

**Review by CriesofCarrots — COMMENTED** [review]
Thank you! Just one suggestion.

File: docs/src/implemented-proposals/staking-rewards.md [hunk 1]
```diff
@@ -30,4 +30,4 @@ Solana's trustless sense of time and ordering provided by its PoH data structure
...
+Similarly, we may consider an algorithmic reduction in a validator's active amount staked in the case that they are offline. I.e. if a validator is inactive for some amount of time, either due to a partition or otherwise, the amount of their stake that is considered ‘active’ \(eligible to earn rewards\) may be reduced. This design would be structured to help long-lived partitions to eventually reach finality on their respective chains as the % of non-voting total stake is reduced over time until a supermajority can be achieved by the active validators in each partition. Similarly, upon re-engaging, the ‘active’ amount staked will come back online at some defined rate. Different rates of stake reduction may be considered depending on the size of the partition/active set.
```
[the remark this answers is not in the document]
Comment (pandabadger) [reply docs/src/implemented-proposals/staking-rewards.md]: Agreed and committed

**Review by CriesofCarrots — APPROVED** [review]
Looks good, thank you!

**Comment by CriesofCarrots** [comment]
Some of these docs were just moved to a different repo. Fixes made in this PR to those files should be reflected here: https://github.com/solana-foundation/developer-content/pull/34

**Review by nickfrosty — APPROVED** [review]
Thanks for the fixes!

**Pull request merged**